pyhifiberry/audiocontrol2sio.py

```diff
@@ -0,0 +1,141 @@
+"""Socketio based API to Hifiberry OS - audiocontrol2"""
+import logging
+
+from socketio import AsyncClient, AsyncClientNamespace
+
+from .consts import DEFAULT_HOST, DEFAULT_PORT
+
+_LOGGER = logging.getLogger(__name__)
+
+class Audiocontrol2SIO:
+
+    def __init__(self, sio) -> None:
+        self.connected = False
+        self.metadata = Metadata(sio)
+        self.player = Player(sio)
+        self.volume = Volume(sio)
+        self._sio = sio
+
+    @classmethod
+    async def connect(cls, host=DEFAULT_HOST, port=DEFAULT_PORT, wait_timeout=10):
+        _LOGGER.debug("TRYING TO CONNECT")
+        sio = AsyncClient()
+        audiocontrol = Audiocontrol2SIO(sio)
+        sio.register_namespace(audiocontrol.metadata)
+        sio.register_namespace(audiocontrol.player)
+        sio.register_namespace(audiocontrol.volume)
+
+        @sio.event
+        async def connect():
+            _LOGGER.info("audiocontroller connected to host %s", host)
```

Comment:
lets make this debug

Reply:
@dgomes I would prefer info, because I think connects and disconnects should appear in the log with default logging config.

Reply:
this message is only relevant for debug purposes ;) a normal user couldn't careless to which address did the library connect to.